docs/_docs/security/sandbox.adoc

```diff
@@ -6,26 +6,75 @@ This user-defined code can utilize Java features to get access to host resources
 For example, it can create/update/delete files; create sockets; read/update/delete system properties; use reflection API and so on.
 To restrict user-defined code options you may use the Ignite Sandbox based on the Java Sandbox model.
 
-You can read about the Java Sandbox model link:https://docs.oracle.com/en/java/javase/11/security/java-se-platform-security-architecture.html#GUID-C203D80F-C730-45C3-AB95-D4E61FD6D89C[here].
+You can read about the Java Sandbox model link:https://docs.oracle.com/en/java/javase/11/security/java-se-platform-security-architecture.html#GUID-C203D80F-C730-45C3-AB95-D4E61FD6D89C[here,window=_blank].
 
-There are a few conditions to run user-defined code with restrictions:
+== The Ignite Sandbox activation
 
-- link:/security/authentication[GridSecurityProcessor] is installed;
-- the method _GridSecurityProcessor#sandboxEnabled_ returns true;
-- link:https://docs.oracle.com/javase/8/docs/technotes/guides/security/spec/security-spec.doc6.html#a19349[SecurityManager] is installed.
+There are a few conditions to activate the Ignite Sandbox.
 
+- The first condition is _the SecurityManager must be installed_.
+
+Due to the Ignite Sandbox is based on the Java Sandbox model, you need to have installed the SecurityManager that is an important part of that model.
+The SecurityManager is responsible for checking the security policy currently in effect and perform access control checks.
```

Comment:
Maybe 'The SecurityManager is responsible for checking, which security policy is currently in effect. It also performs access control checks.'

But I think, we don't need 'checks'. Just '...performs access control.' But it's up to you.